implementations/micrometer-registry-statsd/src/main/java/io/micrometer/statsd/StatsdMeterRegistry.java

```diff
@@ -223,9 +218,9 @@ private void prepareUdpClient(Publisher<String> publisher) {
                 .host(statsdConfig.host())
                 .port(statsdConfig.port())
                 .handle((in, out) -> out
-                        .options(NettyPipeline.SendOptions::flushOnEach)
```

Comment:
NOTE: this option is not available in the latest version of Reactor Netty and I believe is now the default behavior.